spring-boot-project/spring-boot/src/main/java/org/springframework/boot/jdbc/DatabaseDriver.java

```diff
@@ -218,6 +219,19 @@ protected Collection<String> getUrlPrefixes() {
 			return Arrays.asList("ch", "clickhouse");
 		}
 
+	},
+
+	/**
+	 * AWS Advanced JDBC Wrapper.
+	 * @since 3.5.0
+	 */
+	AWS_JDBC_WRAPPER(null, "software.amazon.jdbc.Driver") {
+
+		@Override
+		protected Collection<String> getUrlPrefixes() {
+			return List.of("aws-wrapper");
```

Comment:
I've used `List#of` as it seems to be the simplest option, but note that implementations of `#getUrlPrefixes` in this enum are quite inconsistent - there are usages of `Arrays#asList` and `Collections#singleton`.

Perhaps this could be aligned in a separate issue to use `List#of` consistently (`first-timers-only` candidate?).

Reply:
They seem consistent to me, using `Collections#singleton` when there's a single prefix and `Arrays#asList` when there a multiple prefixes. I'll polish to use `Collections#singleton` when merging.